install.sh

```diff
@@ -0,0 +1,212 @@
+    echo "Removing any old copy of LanguageTools Stable in this directory"
+    rm LanguageTool-stable.zip &>/dev/null
+
+    echo "Installing LanguageTools Stable"
+    version=stable
+    RELEASE_URL="https://languagetool.org/download/LanguageTool-stable.zip"
+    curl -l $RELEASE_URL -o LanguageTool-stable.zip
+    DIR=$(unzip LanguageTool-stable.zip | grep -m1 'creating:' | cut -d' ' -f5- )
+    rm -r $DIR &>/dev/null
+    RELEASE=${DIR%/}
+
+    # Getting rid of any old folders with the same name
+    rm -r $RELEASE-$version
+
+    echo "Unzipping"
+    unzip -u LanguageTool-stable.zip
+
+    mv LanguageTool-3.9 "$RELEASE-$version"
+    echo "Cleaning up"
+    rm LanguageTool-stable.zip
+}
+
+build () {
+    # Cloning from GitHub
+    git clone --depth 1 https://github.com/languagetool-org/languagetool.git
+    cd languagetool || exit 2
+
+    # Seeing if maven is installed, and installing it if not
+    detect_maven
+
+    mvn clean test
+    if [ "$specifypackage" = YES ]; then
+        ./build.sh languagetool-$package package
+    else
+        ./build.sh languagetool-standalone package
+        ./build.sh languagetool-wikipedia package -DskipTests
+        ./build.sh languagetool-office-extension package -DskipTests
+    fi
+}
+
+install_maven() {
+    echo "Installing maven . . ."
+    if [[ "$OSTYPE" == "linux-gnu" ]]; then
+        sudo apt-get update -y
+        sudo apt-get install maven
+
+    elif [[ "$OSTYPE" == "darwin"* ]]; then
+        if ! [ -x "$(brew -v)" ]; then
+               /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+        fi
+        brew update
+        brew install maven
+
+    else
+            echo "Error: java is not installed and operating system detection error"
+            echo "   OS type not supported!"
+            echo "   Please install maven yourself or override automatic OS detection with -o <OS> See help for more details."
+    fi
+}
+
+install_java() {
+    echo "Java is not installed"
+    echo "Installing java . . ."
+
+    if [[ "$OSTYPE" == "linux-gnu" ]]; then
+        sudo apt-get update
+        sudo apt-get install java -y
+
+    elif [[ "$OSTYPE" == "darwin"* ]]; then
+        if ! [ -x "$(brew -v)" ]; then
+               /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+        fi
+        brew update
+        brew cask install java
+
+    else
+            echo "Error: java is not installed and operating system detection error"
+            echo "   OS type not supported!"
+            echo "   Please install maven yourself or override automatic OS detection with -o <OS> See help for more details."
+    fi
+}
+
+detect_maven() {
+    if ! [ -x "$(mvn -v)" ]; then
+        install_maven
+    fi
+}
+
+detect_java() {
+    if ! [ -x "$(java -version)" ]; then
+        install_maven
+    fi
+}
+
+postinstall_command () {
+    if [ "$command" = GUI ] || [ "$command" = GUI ]; then
```

Comment:
The || has the same argument. Maybe `standalone`?
I haven't tested it, but seems to be another great tool. Many thanks.